### PR TITLE
The user framework module complies with s3ip sysfs specification

### PR DESCRIPTION
**Pull request by tianshangfei**
Signed-off-by: tianshangfei <tian_shangfei@163.com>

#### Why I did it
The user framework module complies with s3ip sysfs specification

#### How I did it
1、	create a s3ip_sysfs service
2、 the s3ip_sysfs service call the “s3ip_sysfs_tool.sh” to install kernel module and  run s3ip_load.py
3、 s3ip_load.py will parse the s3ip_sysfs_conf.json configuration file and create /sys_switch/ directory

#### How to verify it
A demo driver base on this framework will display the sysfs node wich conform to the s3ip sysfs specification 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "s3ip_syfs_paths": [
+    {
+      "path": "/sys_switch/temp_sensor",
+      "type" : "path",
+      "value" : "/sys/s3ip/temp_sensor",
+      "description": "温度传感器信息"
```
Comment (FuzailBrcm) [inline platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json:7]: These descriptions are being shown in Mandarin. Can you change it to english pls?

File: platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "电压传感器信息"
+    },
+    {
+      "path": "/sys_switch/syseeprom",
+      "type" : "path",
+      "value" : "/sys/s3ip/syseeprom",
+      "description": "ONIE EEPROM"
+    },
+    {
+      "path": "/sys_switch/fan",
```
Comment (FuzailBrcm) [inline platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json:22]: What if a platform drivers are already loaded and path to read fan_speed is 
/sys/bus/i2c/devices/x-00y/fan<>_input

How can we mention this in the JSON file? Can you give an example?

Comment (tianshangfei) [reply platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json:22]: The example you want is shown below,
{
      "path": "/sys_switch/fan/fan1/motor0/speed",
      "type" : "path",
      "value" : "/sys/bus/i2c/devices/x-00y/fan<>_input",
      "description": "the motor0 of fan1 speed information"
    },

**Comment by tianshangfei** [comment]
> Generally speaking, is it possible to link (via softlinks) the S3IP framework SysFS to the SysFS created by the BSP drivers? If yes, how?

Ok. For example, if the LED driver is based on the kernel framework of S3IP, the framework will automatically create the path /sys/s3ip/sysled.
The S3IP specification requires the path to be /sys_switch/sysled.    Use the following configuration files.   The user framework of S3IP will assist create soft connections from /sys/s3ip/sysled to /sys_switch/sysled.
{
"path": "/sys_switch/sysled",
"type" : "path",
"value" : "/sys/s3ip/sysled",
"description": "SYS LED information"
},

The kernel framework helps you create legitimate hardware information attributes.
The user framework helps you reuse the sysfs path and create the soft link, ultimately creating a directory structure that meets the requirements of  the S3IP specification

**Review by FuzailBrcm — APPROVED** [review]
Changes look good to me

**Pull request merged**